src/views/PaymentNetwork.vue

```diff
@@ -131,51 +149,67 @@ watch(paymentTx, () => {
           class="basis-1/3"
         >
           <label
-            class="flex flex-col sm:flex-row border border-skin-border hover:border-skin-text gap-3 rounded-xl p-3 cursor-pointer"
+            class="flex relative flex-col sm:flex-row gap-2 ring-1 ring-skin-border hover:ring-skin-text rounded-xl p-[12px] pl-3 cursor-pointer"
             :class="{
-              '!border-skin-primary': currencyId === data.currency
+              '!ring-skin-primary': currencyId === data.currency
             }"
             @click="setData('currency', currencyId)"
           >
-            <AvatarToken :address="currency.address[1]" size="30" />
-            <div>
-              <div class="text-skin-heading">
-                {{
-                  computePrice(
-                    currencyId,
-                    PLANS[data.plan].unit,
-                    PLANS[data.plan].discount
-                  ).toLocaleString()
-                }}
-              </div>
-              <small class="leading-none">{{ currency.code }}</small>
-            </div>
+            <AvatarToken :address="currency.address[1]" size="18" />
+
+            <small class="leading-none text-skin-heading text-base">{{
+              currency.code
+            }}</small>
+            <span
+              v-if="currencyId === data.currency"
+              :class="['absolute right-0 pr-3']"
+            >
+              <i-ho-check class="text-skin-primary text-sm" />
+            </span>
           </label>
         </li>
       </ol>
     </fieldset>
 
-    <fieldset class="flex flex-col gap-2">
-      <TuneCheckbox
-        :id="'termsAccepted'"
-        :model-value="data.termsAccepted"
-        hint="By proceeding, I understand that completing the network request form is mandatory after payment."
-        @update:model-value="setData('termsAccepted', $event as boolean)"
-      >
-      </TuneCheckbox>
-    </fieldset>
-
-    <TuneButton
-      primary
-      :disabled="loading || !data.termsAccepted"
-      :type="'submit'"
-      :loading="loading"
-      @click.prevent="pay"
+    <TuneCheckbox
+      :id="'termsAccepted'"
+      :model-value="data.termsAccepted"
+      hint="By proceeding, I understand that completing the network request form is mandatory after payment."
+      @update:model-value="setData('termsAccepted', $event as boolean)"
     >
-      Pay
-      {{ formatCryptoCurrency(amount, CURRENCIES[data.currency]) }} for
-      {{ PLANS[data.plan].label }}
-    </TuneButton>
+    </TuneCheckbox>
+
+    <hr />
```

Comment:
Should use flexbox instead of new line

Reply:
This is not a line return, but a line element

Reply:
Oh :D